profiles/common/modules/features/e_library/e_library_core/e_library_core.install

```diff
@@ -115,3 +126,14 @@ function e_library_core_update_7120() {
     );
   }
 }
+
+/**
+ * Set new value for e_library feature in database.
+ *
+ * Variables from comments settings must be removed from hard config.
+ * Recreate the value of the feature in database without the variable.
+ */
+function e_library_core_update_7121() {
```

Comment:
I do understand why the hook is needed however it does not match the ticket description, do we have agreement on use it or not?

Reply:
This function is needed to have the new value of the feature without reverting it. If we don't do it this way, the modified features should be reverted and some of them mustn't. It is because of these hook updates that the PR is now based in 2.6 and not in 2.5